modules/getting-started/pages/sync-api-apisync.adoc

```diff
@@ -1,21 +0,0 @@
-= To Sync an API
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
-:noindex:
-
-When you update a RAML file from Anypoint Studio, you can synchronize those changes to your API version hosted in Anypoint Platform. Assuming you created the placeholder API described in the previous sections, make some small change to the API. Navigate to your `placeholder.raml` file in the Package Explorer, and double-click to open the file in your RAML Editor. For example, edit the RAML to change a description slightly.
-
-== Synchronize your Updated RAML File With Anypoint Platform
-
-Notice that after making the change, the API Sync view and Upload become available for uploading changes:
-
-image::api-sync-view-raml-update.png[]
-
-You can select the `placeholder.raml` file and click Upload to submit your changes to Anypoint Platform. You can also navigate to your API Designer, or Design Center if you have the Crowd release, to check how the updates were pushed.
```

Comment:
-"You can select...."  Why "you can select" here instead of just "Select," in parallel with the previous example instructions?

- "You can also navigate" > "To verify that your changes took effect, navigate to"
To avoid momentary connection to the submit step rather than to verification

Reply:
great comments. Not taking action as we're deleting this topic.